tests/test_mesh.py

```diff
@@ -340,27 +326,33 @@ def arithmetic_statmesh_setup(self):
         self.statmesh_2.mesh.tag_set_data(flux_tag, volumes1, flux_data)
         self.statmesh_2.mesh.tag_set_data(error_tag, volumes2, error_data)
 
-
-    def test_mutliply_vector_by_scalar(self):
+    def test_slice_operator_vector_tag(self):
         self.arithmetic_mesh_vector_setup()
-        scalar = 12.0
-        test_vector_tag = self.mesh_1_vector.mesh.tag_get_handle(self.vector_tag_name)
-        test_data = self.mesh_1_vector.mesh.tag_get_data(test_vector_tag, self.volumes1)
-        exp_res = [[12.0, 24.0], [18.0, 30.0], [-12.0, -18.0], [-24.0, -30.0]]
-        obs_res = scalar*test_data
-        assert_array_almost_equal(exp_res, obs_res)
-    
-    def test_multiply_vector_tag_by_scalar_tag(self):
+        exp_res = [[1.0, 2.0], [1.5, 2.5], [-1.0, -1.5], [-2.0, -2.5]]
+        mesh_1_vector_data = self.mesh_1_vector.testing[:]
+        assert_array_almost_equal(exp_res, mesh_1_vector_data)
+        mesh_2_vector_data = self.mesh_2_vector.testing[:]
+        index = 0
+        exp_res = [[[15.0, 50.0], [10.0, 40.0], [-15.0, -50.0], [-20.0, -50.0]], 
+                   [[22.5, 62.5], [15.0, 50.0], [-22.5, -62.5], [-30.0, -62.5]], 
+                   [[-15.0, -37.5], [-10.0, -30.0], [15.0, 37.5], [20.0, 37.5]], 
+                   [[-30.0, -62.5], [-20.0, -50.0], [30.0, 62.5], [40.0, 62.5]]]
+
+        for result in exp_res:
+            assert_array_almost_equal(result, mesh_1_vector_data[index]*mesh_2_vector_data)
+            index += 1
```

Comment:
I'm not sure we need to test this.  Once we have confirmed that the slice operators behave correctly, this is effectively the same as testing numpy's operators, isn't it?  We probably should assume that those are well tested already.

Or am I missing something?

Reply:
Also - I think the tests for the slice operator may already exist in `test_nativtag()` and methods that come after that on about line 763

Reply:
Looking at the `test_nativetag` and related methods, I agree that this functionality is already being tested. You mentioned above that we should look at doing some fancy indexing with the tags, but I think that is tested as well in the `test_nativetag_fancy_indexing()` method. So maybe move on to make some operators? They would be in mesh.py probably?

Reply:
Also, I don't really know what an operator implementation would look like. Is it similar to the mesh functions that I was looking at related to `_do_op()`?

Reply:
I think `_do_op()` is a pattern we may eventually want to follow, but you could start with just implementing a single operator that is a member function of the `NativeTag` class: e.g. `def mult(self, tag)` that invokes a slice on `self` and a slice on `tag` and multiplies them.  It would need to check a bunch of things to make sure it was safe/robust to different variations like:
* scalar * vector_tag
* vector * scalar_tag
* vector * vector_tag
* vector * scalar_value
* scalar * vector_value

etc